src/core/build_target.go

```diff
@@ -247,7 +247,8 @@ type BuildMetadata struct {
 	// Standard output & error
 	Stdout, Stderr []byte
 	// Serialised build action metadata.
-	RemoteAction []byte
+	RemoteAction  []byte
+	RemoteOutputs []byte
```

Comment:
I wonder if this stuff needs to be stored, or if it could just be read back from disk on demand?